src/opm/output/eclipse/InteHEAD.cpp

```diff
@@ -343,10 +343,10 @@ enum index : std::vector<int>::size_type {
   ih_285       =      285       ,              //       0
   MAX_ANALYTIC_AQUIFERS= VI::intehead::MAX_ANALYTIC_AQUIFERS, //  Declared maximum number of analytic aquifers in model.  AQUDIMS(5).
   ih_287       =      287       ,              //       0
-  ih_288       =      288       ,              //       0
-  ih_289       =      289       ,              //       0
-  NOIUADS       =      VI::intehead::NO_IUADS,  //       0
-  NOIUAPS       =      VI::intehead::NO_IUAPS,  //       0
+  NUM_AQU_UDQS =      VI::intehead::NO_AQU_UDQS,      //       0
+  NUM_BLK_UDQS =      VI::intehead::NO_AQU_UDQS,      //       0
```

Comment:
I am not into this code, but this strikes me. Is it on purpose that both entries get assigned `NO_AQU_UDQS`?

Reply:
>  Is it on purpose that both entries get assigned `NO_AQU_UDQS`?

Good catch!  No, that's not on purpose&ndash;it's a mistake.  I've pushed an update to correct the mistake.